addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -448,8 +450,14 @@ export class Wysiwyg extends Component {
             getPowerboxElement: () => {
                 const selection = (this.options.document || document).getSelection();
                 if (selection.isCollapsed && selection.rangeCount) {
-                    const baseNode = closestElement(selection.anchorNode, 'P:not([t-field]), DIV:not([t-field]):not(.o_not_editable)');
-                    const fieldContainer = closestElement(selection.anchorNode, '[data-oe-field]');
+                    const deepestNode = getDeepestPosition(selection.anchorNode, selection.anchorOffset);
+                    const elementSelectors = [
+                        'LI:not([t-field])',
+                        'DIV:not([t-field]):not(.o_not_editable)'
+                    ];
+                    elementSelectors.push(...paragraphRelatedElements.map(element => `${element}:not([t-field])`));
+                    const baseNode = closestElement(deepestNode[0], elementSelectors);
```

Comment:
I was very surprised to see this work because I could not find it documented anywhere, but I tried on Chrome, Firefox and Safari and apparently `closest` does support receiving an array as parameter. It probably works by luck because of type coercion since `String(['P', 'DIV'])` equals `'P,DIV'`. That being said, I was so taken aback by this that I think it would be better to make the conversion explicit by doing `elementSelectors.join(',')` when calling `closestElement`.